environments/Readme.md

```diff
@@ -1,10 +1,10 @@
 # momepy container
 
-`momepy` is a library for quantitative analysis of urban form - urban morphometrics - built on top of GeoPandas, PySAL and networkX. This container provides the minimal environment needed to run momepy. 
+`momepy` is a library for quantitative analysis of urban form - urban morphometrics - built on top of GeoPandas, PySAL and networkX. This container provides the minimal environment needed to run momepy.
```

Comment:
Should this text reflect that in `docs/index.rst`?

```
It is part of `PySAL (Python Spatial Analysis Library)`_
 and is built on top of `GeoPandas`_, other `PySAL`_ modules and
 `networkX`_.
```

Reply:
Yes, forgot about this bit. And it reminds me that the docker container is out of sync with the latest release.